docs/visualization/plotting-overview.ipynb

```diff
@@ -166,7 +163,7 @@
    "outputs": [],
    "source": [
     "d.coords['xlabels'] = sc.Variable(['x'], values=np.arange(M) + 15.)\n",
-    "plot(d['data_2d'], axes=['y', 'xlabels'])"
+    "plot(d['data_2d'], axes={'x': 'xlabels'})"
    ]
   },
```

Comment:
Not necessarily related to what has been done here, but `help(plot)` is relatively uninformative as opposed to running `help(plot1d)`, since in the documentation we are encouraging people to use the top level `plot`, could we expand the help for `plot` to at least point people to the sub-plotting commands.

Reply:
ah yes, that was one of Simon's comments also. I need to add docstrings to everything.
maybe i will do this as a follow up